DaysOfCode/classVsInstance.js

```diff
@@ -1,60 +1,71 @@
+/* eslint-disable no-unused-vars */
 /* eslint-disable prefer-const */
 /* eslint-disable radix */
-
 let inputStdin = '';
 let inputStdinArray = '';
 let inputCurrentline = 0;
 
 process.stdin.resume();
 process.stdin.setEncoding('ascii');
 process.stdin.on('data', (data) => {
-    inputStdin += data;
+  inputStdin += data;
 });
 
 process.stdin.on('end', () => {
-    inputStdinArray = inputStdin.split('\n');
-    main();
+  inputStdinArray = inputStdin.split('\n');
+  main();
 });
 
 function readLine() {
-    return inputStdinArray[inputCurrentline += 1];
+  return inputStdinArray[inputCurrentline += 1];
 }
 
 function Person(initialAge) {
-    let age = 0;
-    // Add some more code to run some checks on initialAge
-    if (initialAge < 0) {
-        console.log('Age is not valid, setting age to 0.');
-        age = 0;
-    } else {
-        age = initialAge;
+  const age = 0;
+
+  function setAge(newAge) {
```

Comment:
I am not a big fan of declaring functions inside functions please take this functions away., and have smaller functions, also please configure your eslint to not allow you to have you functions with more than 40 lines of code..